controllers/chat/sendSignedMessage.js

```diff
@@ -139,11 +139,17 @@ const sendSignedMesage = async (req, res) => {
       });
     }
 
-    const chat = await channel.sendMessage(baseMessage, {skip_push: true});
+    let chat = await channel.sendMessage(baseMessage, {skip_push: true});
     let notificationPayload = {
       title: `${senderInfo?.user?.username}`,
       body: `${message.substring(0, 100)}`
     };
+
+    if (chat && chat.message?.message_type === MESSAGE_TYPE.TEXT) {
+      chat.message.attachments = [];
+      attachments = [];
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `const` to `let` for `chat` is justified if you're modifying its properties later in the code. However, the conditional block introduced here seems to clear the attachments of a text message. This could potentially remove important data. Ensure that this operation is intended and won't lead to data loss. Also, consider adding error handling for the case when `chat.message` is undefined to prevent potential runtime errors.
```diff
- if (chat && chat.message?.message_type === MESSAGE_TYPE.TEXT) {
-   chat.message.attachments = [];
-   attachments = [];
- }
+ if (chat && chat.message?.message_type === MESSAGE_TYPE.TEXT) {
+   if(chat.message.attachments){
+     chat.message.attachments = [];
+   } else {
+     throw new Error('Chat message attachments are undefined');
+   }
+   if(attachments){
+     attachments = [];
+   } else {
+     throw new Error('Attachments are undefined');
+   }
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->